### PR TITLE
Prevent last page number from showing on empty folder

### PR DESCRIPTION
**Pull request by thanhz**
The navigation page was showing '0' when navigating into an empty folder

### CODE REVIEW

**Review by vimleshtna — COMMENTED** [review]
Looks good however If possible, can we add a test for this, please?
Not sure, if we have an end-to-end test to cover this scenario.

**Comment by thanhz** [comment]
> Looks good however If possible, can we add a test for this, please? Not sure, if we have an end-to-end test to cover this scenario.

Thanks, I've added more cases to the empty folder test. I believe we won't be adding e2e tests until all the pages are linked

File: app/views/standard/additionalMetadataNavigation.scala.html [hunk 1]
```diff
@@ -96,7 +96,7 @@ <h2 class="app-task-list__section">Folder uploaded: @folderUploaded</h2>
                                 @if(totalPages - currentPage > 3) {
                                         <li class="govuk-pagination__item govuk-pagination__item--ellipses">&ctdot;</li>
                                 }
-                            @if(totalPages != 1) {
+                            @if(totalPages != 1 && totalPages != 0) {
```
Comment (vimleshtna) [inline app/views/standard/additionalMetadataNavigation.scala.html:99]: Can we add one check which will work for both conditions: `@if(totalPages > 1)`?

**Review by vimleshtna — COMMENTED** [review]
Look good, just a question, thanks

File: app/views/standard/additionalMetadataNavigation.scala.html (hunk 1 above)
Comment (thanhz) [reply app/views/standard/additionalMetadataNavigation.scala.html]: Yes you're right, that is much simpler :+1:

**Review by vimleshtna — APPROVED** [review]
Looks good, thanks,

**Pull request merged**